NuGet.Config

```diff
@@ -2,15 +2,13 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="NuGet.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+    <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
+    <add key="myget-legacy@Local" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy%40Local/nuget/v3/index.json" />
     <add key="nuget-build" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json" />
+    <add key="NuGet.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
does the order of sources matter ? I see that `NuGet.org` has been moved from the top. If there is any doc where I can learn more , I will be happy to do that.

Reply:
no, order doesn't matter. I just used VS Code's "sort lines" feature.